dataactcore/scripts/updateCGAC.py

```diff
@@ -20,21 +21,44 @@ def updateCGAC():
     all_users = userDb.session.query(User).all()
     all_submissions = jobDb.session.query(Submission).all()
 
+    users_to_update = []
+
     for user in all_users:
-        user.cgac_code = findCGACCode(all_cgac, user.cgac_code)
+        if user.cgac_code != None:
+            users_to_update.append(user)
 
-    for submission in all_submissions:
-        submission.cgac_code = findCGACCode(all_cgac, submission.cgac_code)
+    for user in users_to_update:
+        if user.user_id == 2:
+            if validAgency(user.cgac_code, validationDb):
+                user.cgac_code = validationDb.getCGACCode(user.cgac_code)
```

Comment:
As written this is calling getCGACCode twice, could we just have validAgency pass back the cgac code to be stored here if it is not None, so that we don't have to do each query twice?  Probably not that important since this script isn't something that needs to be really efficient.